### PR TITLE
documentation of component library

### PR DESCRIPTION
**Pull request by ravikumar1002**
documentation for every component
home page of component
introduction page

### CODE REVIEW

File: docs/docs_modal/docs_modal.js [hunk 1]
```diff
@@ -0,0 +1,10 @@
+let modalShowBtn = document.querySelectorAll(".modal-demo-btn");
+let modalShowDemo = document.querySelectorAll(".modal-demo-result");
```
Comment (romabulani) [inline docs/docs_modal/docs_modal.js]: You can use const here

Comment (ravikumar1002) [reply docs/docs_modal/docs_modal.js]: I changed it
Thanks for reviewing

File: components/snackbar/snackbar.html [hunk 2]
```diff
@@ -1,54 +1,177 @@
...
+          src="https://carbon.now.sh/embed?bg=rgba%28255%2C255%2C255%2C1%29&t=one-light&wt=none&l=javascript&width=572.5&ds=true&dsyoff=0px&dsblur=2px&wc=true&wa=true&pv=8px&ph=14px&ln=false&fl=1&fm=Hack&fs=14px&lh=167%25&si=false&es=2x&wm=false&code=const%2520snackbarRemove%2520%253D%2520%2520document.querySelector%28%2522.snackbar%2522%29%250A%250AsetTimeout%28%28%29%2520%253D%253E%2520%257B%250A%2520%2520%2520%2520%2520snackbarRemove.classList.add%28%2522snackbar-remove%2522%29%250A%2520%257D%252C%25201000%29%253B%250A"
+          sandbox="allow-scripts allow-same-origin">
+        </iframe>
+        </code>
+      </div>
+    </section>
+    </div>
+  </main>
+  <script src="../../docs/docs-css/main.js"></script>
+  <!-- <script src="./snackbar.js"></script> -->
```
Comment (romabulani) [inline components/snackbar/snackbar.html]: You can remove line if not in use

**Review by romabulani — APPROVED** [review]
- Introduction.css is empty, you can delete it
- Write good description while raising PR, add gif if possible
- Try to use prettier/beautify, inside aside tag, span tag for navigation is on same line

Rest all looks good to me, good work, you can merge

**Review by shrey27 — APPROVED** [review]

**Comment by ravikumar1002** [comment]
> * Introduction.css is empty, you can delete it
> * Write good description while raising PR, add gif if possible
> * Try to use prettier/beautify, inside aside tag, span tag for navigation is on same line
> 
> Rest all looks good to me, good work, you can merge

 I used pretty, but sometimes it works and sometimes it doesn't work.
yes I  have to remove some file,
And I will take care of the description from next time.

**Pull request merged**